docs/index.html

```diff
@@ -1,41 +1,39 @@
 <!DOCTYPE html>
-<html >
-<head>
-  <meta charset="UTF-8">
-  <title>Flooper.js</title>
-
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
-
-  <link rel="stylesheet" href="style.css">
-
-
-</head>
-<body>
-  <div class="o-skeleton">
-    <h1>Flooper.js Demo</h1>
-
-    <main class="o-skeleton__body">
-      <!--content here-->
-      <div class="c-infinity">
-        <div class="c-infinity__row js-flooper">
-          <div class="js-flooper-item c-infinity__block c-infinity__block--2x">1</div>
-          <div class="js-flooper-item c-infinity__block">2</div>
-          <div class="js-flooper-item c-infinity__block">3</div>
-          <div class="js-flooper-item c-infinity__block c-infinity__block--2x">4</div>
-          <div class="js-flooper-item c-infinity__block">5</div>
-          <div class="js-flooper-item c-infinity__block c-infinity__block--2x">block</div>
-          <div class="js-flooper-item c-infinity__block">6</div>
-          <div class="js-flooper-item c-infinity__block">7</div>
-          <div class="js-flooper-item c-infinity__block">8</div>
-          <div class="js-flooper-item c-infinity__block c-infinity__block--2x">9</div>
-          <div class="js-flooper-item c-infinity__block">10</div>
-          <div class="js-flooper-item c-infinity__block">11</div>
-
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="Desc">
+    <meta name="keywords" content="gulp, babel, sass, jade, javaScript">
+    <title>Flooper.js</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0,user-scalable=no">
```

Comment:
Wouldn't it be nice, if one day... you know... we didn't had to add 100000000 lines in the head tag every time we create a new doc? 🤔 💭

Reply:
We can, we are using **Jade** templating language. `docs/*` are generated files. Everything comest from `./site` folder!

Bonus tip: all meta info comes from `jade/config.json`, that are expose to `.jade` files as variables.
Bonus tip2: it supports partials, just go to `site/jade/` and create `_head.jade`. 🎩